src/geo/leaflet/leaflet-cartodb-layer-group-view.js

```diff
@@ -219,23 +207,15 @@ var LeafletCartoDBLayerGroupView = L.TileLayer.extend({
    * generates a new url for tiles and refresh leaflet layer
    * do not collide with leaflet _update
    */
-  __update: function (done) {
-    var self = this;
-    this.fire('updated');
-    this.fire('loading');
-    var map = this.options.map;
-
-    var tilejson = self.model.get('urls');
+  __update: function () {
```

Comment:
Is there a reason why this has to be called `__update`? Otherwise, I would call it `onTileJSONChanged`...